packages/video-intelligence/README.md

```diff
@@ -1,19 +1,19 @@
-# Node.js Client for Google Cloud Video Intelligence API ([Alpha](https://github.com/GoogleCloudPlatform/google-cloud-node#versioning))
+# Node.js Client for Google Cloud Video Intelligence API ([Beta](https://github.com/GoogleCloudPlatform/google-cloud-node#versioning))
```

Comment:
I just wasn't sure that meant that our library needed to mirror the version of an upstream API. So far, it's been determined by the stability of our own API.